WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -185,8 +185,21 @@ def getCPUNormalization( reference = 'HS06', iterations = 1 ):
 
 
 def getCPUTime( cpuNormalizationFactor ):
-  """ Trying to get CPUTime (in seconds) from the CS. The default is a large 9999999, that we may consider as "Infinite".
+  """ Trying to get CPUTime left for execution (in seconds).
+
+      It will first look to get the work left looking for batch system information useing the TimeLeft utility.
+      If it suceeds, it will convert it in real second, and return it.
+
+      If it fails, it tries to get it from the static info found in CS.
+      If it fails, it returns the default, which is a large 9999999, that we may consider as "Infinite".
```

Comment:
we could use ```sys.maxint``` for all these 'infinite' values. I mean, not only here, but throughout DIRAC